scripts/env-doctor.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -e
+
+# Check if running on macOS
+if [[ "$(uname)" != "Darwin" ]]; then
+  echo "Error: This script only works on macOS."
+  echo "Your current operating system is: $(uname)"
+  exit 1
+fi
```

Comment:
Why is this MacOS only? A decent chunk of us are on Linux

Reply:
Because some remedies use Homebrew. Also Linux uses are less likely to need it. ;-) But feel free to improve the script.